client/jetpack-cloud/sections/partner-portal/stored-credit-card/style.scss

```diff
@@ -20,9 +20,32 @@
 
 .stored-credit-card__header {
 	display: flex;
-	justify-content: space-between;
-	flex-grow: 1;
-	margin-bottom: auto;
+	align-items: flex-start;
+	flex-basis: 100%;
+}
+
+.stored-credit-card__labels {
+	display: flex;
+
+	> div {
+		&:first-child {
+			margin-left: 0;
+		}
+
+		margin-left: 10px;
+		margin-right: 10px;
+	}
+}
+
+.stored-credit-card__actions {
+	width: 28px;
+	height: 28px;
+
+	margin-left: auto;
```

Comment:
Another way of accomplishing this (pushing the actions element to the right), would be to add `justify-content: space-between;` to `.stored-credit-card__header`, which makes the children of that element to be pushed to the sides of it, and the rest of the space to be included in the middle of the children.

Reply:
BTW, it's totally fine to do it the way you did it. I only mention it to show there are other ways of accomplishing the same goal.